pinot-plugins/pinot-minion-tasks/pinot-minion-builtin-tasks/src/main/java/org/apache/pinot/plugin/minion/tasks/mergerollup/MergeRollupTaskGenerator.java

```diff
@@ -260,9 +270,13 @@ public List<PinotTaskConfig> generateTasks(List<TableConfig> tableConfigs) {
         // Get watermark from MergeRollupTaskMetadata ZNode
```

Comment:
May be this comment goes into the if now?

Reply:
Yep, updated.